oem.php

```diff
@@ -13,19 +13,19 @@
 
 <div class="grid">
     <div class="two-thirds">
-        <h1>Information for OEMs</h1>
+        <h1>Info for OEMs</h1>
         <h4>Give your customers the best experience with elementary OS.</h4>
     </div>
 </div>
 
 <div class="grid">
     <div class="two-thirds">
-        <h2>Partners &amp; Retailers</h2>
+        <h2>Partners and Retailers</h2>
         <p>We offer two ways to be listed as an OEM of elementary OS in <a href="https://store.elementary.io">our store</a>, with differing requirements and benefits.</p>
     </div>
     <div class="half copy">
         <h3>Partner</h3>
-        <p>Model-by-model approval with those specific models featured in <a href="https://store.elementary.io">our store</a>.</p>
+        <p>Model-by-model approval, with each of them featured in <a href="https://store.elementary.io">our store</a>.</p>
```

Comment:
This is clunkier to me, though I agree the original text may not have been the best. How about:

```suggestion
        <p>Per-model approval, with each model featured in <a href="https://store.elementary.io">our store</a>.</p>
```